### ISSUE TITLE
[BUG] Formatting issue and typo in help for element_mouseevent_keyboard 

### ISSUE THREAD

**Issue by aliunwala** [IBMa/equal-access#941]
## What tool(s) are you using?

- [x] `accessibility-checker-engine`
- [x] `accessibility-checker`

## Detailed description

> Platform: Mac Big Sur, Chrome

> Help contains a table that is not rendered with the correct font or clear table structure.  
> New 2023-10-24: that text color in the "table" remains black and therefore fails in one of the dark modes
> There is an extra ')' in the 'About this requirement' section.

- ws the old E2E issue #3153

## Steps to reproduce the issue

1. Open the test page: equal-access/accessibility-checker-engine/test/v2/checker/accessibility/rules/Rpt_Aria_EventHandlerMissingRole_Native_Host_Sematics_ruleunit/eventHandlerMissingRole2.html
2. Run the checker on this page
3. Find the issue 'Confirm the <div> element with mouse event handler(s) 'onclick' has a corresponding keyboard handler(s)'
4. Open the help for this issue - it contains ~~two~~ three (3)problems: 
 * a table that is not formatted, the text color doesn;t honor dark mode, and 
 * a stray ')' in the 'About this requirement' section

## Additional information
![Screen Shot 2021-05-27 at 8.05.18 PM.png](https://zenhub.ibm.com/images/5821294ffdfd5b9223bb7e4d/370795ef-b072-4e52-b334-6cfb7e1a1046)

——————
As of June 1
- [x] general rule description (SME & Dev)
- [x] testcases written (Dev)
- [x] rule developed (Dev)
- [x] message written (SME)
- [x] help written (SME)

![image](https://user-images.githubusercontent.com/4451128/172454443-0936eb94-60a8-4263-ac82-169e169f7305.png)

**Comment by aliunwala** [comment]
Triage: 
Making this dev since we have a table formatting error.

**Comment by marcjohlic** [comment]
@philljenkins do you have an update on this one?

**Comment by philljenkins** [comment]
The rule was re-named to 
**_[element_mouseevent_keyboard](https://unpkg.com/accessibility-checker-engine@3.1.62/help/en-US/element_mouseevent_keyboard.html#%7B%22message%22%3A%22Confirm%20the%20%3C%7B0%7D%3E%20element%20with%20mouse%20event%20handler(s)%20'%7B1%7D'%20has%20a%20corresponding%20keyboard%20handler(s)%22%2C%22msgArgs%22%3A%5B%22%7B0%7D%22%2C%22%7B1%7D%22%2C%22%7B3%7D%22%2C%22%7B4%7D%22%5D%2C%22value%22%3A%5B%22RECOMMENDATION%22%2C%22MANUAL%22%5D%2C%22reasonId%22%3A%22Manual_1%22%7D)_**: All interactive content with mouse event handlers must have equivalent keyboard access

and still has a problem displaying the data table
![image](https://github.com/IBMa/equal-access/assets/16193609/5f0edfb4-b097-440e-8bf6-9eae300d0554)

**Comment by philljenkins** [comment]
The "extra parentheses" is still there in the first bullet, but that is a very simple fix
![image](https://github.com/IBMa/equal-access/assets/16193609/1f76be49-d07a-4bb6-a423-2e54dd6ac685)

**Comment by philljenkins** [comment]
However, when the Help is displayed inside the Checker UI, the black text color is transformed correctly to white, however, the formatting is still odd.:
![image](https://github.com/IBMa/equal-access/assets/16193609/61c95fcf-ad4b-4ecb-aea7-93b3ca763d4b)

The MDX source code
![image](https://github.com/IBMa/equal-access/assets/16193609/57d6f650-aaa6-4514-b901-1e19e6a35056)

**Comment by philljenkins** [comment]
Some changes, such as removing the ")" have been made in dev-1674 branch / PR 1719
   accessibility-checker-engine/help-v4/en-US/element_mouseevent_keyboard.html

https://github.com/IBMa/equal-access/pull/1719/files#diff-e4d9dfd0b8cee30762cc00be33288cea1cbabf4b78123e3775df5815cf742131

**Comment by philljenkins** [comment]
assigning to Tom to fix the CSS both in Checker and Rule sets web page.

**Comment by tombrunet** [comment]
@philljenkins I put the formatting issues into a separate PR since I needed to touch all of the help files to make sure they had the doctype declaration. See https://github.com/IBMa/equal-access/pull/1730. Image with mouse hover of the second row.

<img width="611" alt="image" src="https://github.com/IBMa/equal-access/assets/19577713/5292f30a-4fac-4ecb-9288-df267ab8747e">

**Comment by tombrunet** [comment]
Also, the only help file I could find with a table is `element_mouseevent_keyboard.html`.

**Comment by philljenkins** [comment]
I reviewed & approved the PR #1730

**Comment by philljenkins** [comment]
@ErickRenteria the PR #1730 can be tested by running the ruleunit testcase and displaying the Help to see the table formatted in the Help page
`... equal-access/accessibility-checker-engine/test/v2/checker/accessibility/rules/element_mouseevent_keyboard_ruleunit/Events-invalidNoMouseRequired.svg`

Also note that building all PR's now generates a `Rule listing` artifact in the Actions (see screenshot) that lists the rules, showing the new `WCAG 2.2 (A, AA)` option and the new WCAG 2.2 rules, such as 
> 2.4.11 Focus Not Obscured (Minimum) [AA]
When an element receives focus, it is not entirely covered by other content.
element_tabbable_unobscured: When an element receives focus, it is not entirely covered by other content

![image](https://github.com/IBMa/equal-access/assets/16193609/7b4a9b82-d311-413b-8293-ff6d7bc258cf)

![image](https://github.com/IBMa/equal-access/assets/16193609/1e784383-46bd-4101-9cd6-b70f83187e19)

**Comment by philljenkins** [comment]
The earlier fixes are in the PR #1719

**Comment by philljenkins** [comment]
@ErickRenteria this issue can be tested by running the ruleunit testcase and displaying the Help to see the table formatted correctly in the Help page
`... equal-access/accessibility-checker-engine/test/v2/checker/accessibility/rules/element_mouseevent_keyboard_ruleunit/Events-invalidNoMouseRequired.svg`

Also note that building all PR's now generates a `Rule listing` artifact in the Actions (see screenshot) that lists the rules, showing the new `WCAG 2.2 (A, AA)` option and the new WCAG 2.2 rules, such as 
> 2.4.11 Focus Not Obscured (Minimum) [AA]
When an element receives focus, it is not entirely covered by other content.
element_tabbable_unobscured: When an element receives focus, it is not entirely covered by other content

![image](https://github.com/IBMa/equal-access/assets/16193609/7b4a9b82-d311-413b-8293-ff6d7bc258cf)

![image](https://github.com/IBMa/equal-access/assets/16193609/1e784383-46bd-4101-9cd6-b70f83187e19)
